doc/content/verification_and_validation/val-2c.md

```diff
@@ -0,0 +1,148 @@
+The first approach, like [!citep](Holland1986,longhurst1992verification), assumes that the entire tritium inventory is immediately injected in the enclosure at the beginning of the experience.
+The second approach assumes that the tritium inventory is being injected into the enclosure at a linear rate during a period of time $t_injection$ until the entire tritium inventory is injected.
+The results of these two approaches are presented and discussed below.
+
+
+## Case and Model Parameters
+
+The case and model parameters used in both approaches in TMAP8 are listed in [val-2c_parameters]. Some of the parameters are directly leveraged from [!cite](Holland1986,longhurst1992verification,ambrosek2008verification), but others were adapted to better match the experimental data.
+
+
+!table id=val-2c_parameters caption=Case and model parameters values used in both approaches in TMAP8 with $R$, the gas constant, and $N_A$, Avogadro's number, as defined in [PhysicalConstants](source/utils/PhysicalConstants.md). When values are the same for both approaches, they are noted as identical. Units are converted in the input file.
+| Parameter                  | Immediate injection approach                    | Delayed injection approach                      | Unit       | Reference                          |
+| -------------------------- | ----------------------------------------------- | ----------------------------------------------- | ---------- | ---------------------------------- |
+| $V$                        | 0.96                                            | Identical                                       | m$^3$      | [!cite](Holland1986)               |
+| $l$                        | 0.16 (between 0.1 and 0.2)                      | Identical                                       | mm         | [!cite](Holland1986)               |
+| T$_2^0$                    | 10                                              | Identical                                       | Ci/m$^{3}$ | [!cite](Holland1986)               |
+| $c_{\text{H}_2\text{O}}^0$ | 714                                             | Identical                                       | Pa         | [!cite](longhurst1992verification) |
+| $f$                        | 0.54                                            | Identical                                       | m$^3$/hr   | [!cite](Holland1986)               |
+| $T$                        | 303                                             | Identical                                       | K          | [!cite](Holland1986)               |
+| Total time                 | 180000                                          | Identical                                       | s          | [!cite](Holland1986)               |
+| $K^0$                      | $1.5 \times 2.0 \times 10^{-10}$                | $2.8 \times 2.0 \times 10^{-10}$                | m$^3$/Ci/s |                                    |
```

Comment:
Why is $K^0$ formatted in this way instead of simply $3.0 \times 10^{-10}$? Similar question for the $K_s^i$ quantities below.

Reply:
If this is showing the literature quantity and a multiplicative factor, it might need to be displayed a bit more clearly.

Reply:
That's what it is. The values from TMAP4/7 are listed with the corrective factor before it. I think I should just put the final value instead, or maybe put the corrective factor in italic, or something.

Reply:
I put it in bold.